tensortrade/instruments/quantity.py

```diff
@@ -59,11 +63,30 @@ def path_id(self) -> str:
     def path_id(self, path_id: str):
         self._path_id = path_id
 
+    @property
+    def src(self):
```

Comment:
`src` and `tgt` should be `source` and `target`, just because we don't really abbreviate anything else in the library.

Reply:
Sounds good to me, but it's only really used for logging purposes. I'm fine either way.